glib-macros/Cargo.toml

```diff
@@ -19,6 +19,7 @@ proc-macro2 = "1.0"
 quote = "1.0"
 syn = "1.0"
 proc-macro-crate = "0.1"
+bitflags = "1.0"
```

Comment:
Do we have to depend on it if we only expand into code that uses the macro? The user of our macro will have to depend on bitflags AFAIU.

Or are you re-exporting it and using the re-exported version from the generated code?